khmer/khmer_args.py

```diff
@@ -504,8 +504,14 @@ def calculate_graphsize(args, graphtype, multiplier=1.0):
         raise ValueError('unknown graph type: ' + graphtype)
 
     if args.max_memory_usage:
+        # subtract about 30MB for the python interpreter if user is requesting
+        # a large amount of memory that suggests they want to limit the
+        # total amount of RAM used for purposes of not running out of memory
+        max_memory = args.max_memory_usage
+        if max_memory > 1e8:
+            max_memory -= 3e7
         tablesize = (khmer._buckets_per_byte[graphtype] *
-                     args.max_memory_usage / args.n_tables / float(multiplier))
+                     max_memory / args.n_tables / float(multiplier))
```

Comment:
Bucket size multiplier maybe? The multiplier parameter certainly predates any of my hacking on memory related arguments.